powershell/resources/built-time-cmdlets.csproj

```diff
@@ -11,32 +11,32 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Compile Include="..\..\csharp-v2\resources\runtime\csharp\client\IAssociativeArray.cs" Link="IAssociativeArray.cs" />
-    <Compile Include="..\..\csharp-v2\resources\runtime\csharp\client\InfoAttribute.cs" Link="InfoAttribute.cs" />
-    <Compile Include="runtime\BuildTime\Cmdlets\ExportModelSurface.cs" />
-    <Compile Include="runtime\BuildTime\Cmdlets\ExportCmdletSurface.cs" />
-    <Compile Include="runtime\BuildTime\Cmdlets\ExportHelpMarkdown.cs" />
-    <Compile Include="runtime\BuildTime\Cmdlets\ExportFormatPs1xml.cs" />
-    <Compile Include="runtime\BuildTime\Cmdlets\ExportExampleStub.cs" />
-    <Compile Include="runtime\BuildTime\Cmdlets\GetModuleGuid.cs" />
-    <Compile Include="runtime\BuildTime\Cmdlets\ExportPsd1.cs" />
-    <Compile Include="runtime\BuildTime\Cmdlets\GetCommonParameter.cs" />
-    <Compile Include="runtime\BuildTime\Models\PsHelpMarkdownOutputs.cs" />
-    <Compile Include="runtime\BuildTime\Models\PsMarkdownTypes.cs" />
-    <Compile Include="runtime\BuildTime\Models\PsHelpTypes.cs" />
-    <Compile Include="runtime\BuildTime\Models\PsFormatTypes.cs" />
-    <Compile Include="runtime\BuildTime\Cmdlets\GetScriptCmdlet.cs" />
-    <Compile Include="runtime\BuildTime\Cmdlets\ExportProxyCmdlet.cs" />
-    <Compile Include="runtime\BuildTime\Cmdlets\ExportTestStub.cs" />
-    <Compile Include="runtime\BuildTime\MarkdownRenderer.cs" />
-    <Compile Include="runtime\BuildTime\PsHelpers.cs" />
-    <Compile Include="runtime\BuildTime\PsAttributes.cs" />
-    <Compile Include="runtime\BuildTime\PsExtensions.cs" />
-    <Compile Include="runtime\BuildTime\Models\PsProxyOutputs.cs" />
-    <Compile Include="runtime\BuildTime\Models\PsProxyTypes.cs" />
-    <Compile Include="runtime\BuildTime\CollectionExtensions.cs" />
-    <Compile Include="runtime\BuildTime\StringExtensions.cs" />
-    <Compile Include="runtime\BuildTime\XmlExtensions.cs" />
+    <Compile Include="runtime\csharp\client\IAssociativeArray.cs" Link="IAssociativeArray.cs" />
+    <Compile Include="runtime\csharp\client\InfoAttribute.cs" Link="InfoAttribute.cs" />
+    <Compile Include="psruntime\BuildTime\Cmdlets\ExportModelSurface.cs" />
```

Comment:
why not leave these in the runtime directory?  Also, do any of the scripts depend on the current path?

Reply:
Garrett refactor the code structure, which leads to this issue. I just fix the reference issue after code structure changes.